### DISCUSSION TITLE
Unable to login on Arch Linux KDE using Microsoft Edge

### DISCUSSION THREAD

**Discussion by RoastBeefer00** [deta/cloud-docs#395] [Help]
Upon using "deta login" command I receive the following:

Please, log in from the web page. Waiting...
Error: lookup localhost on 1.1.1.1:53: no such host

My browser doesn't open any page and I am unable to log in.

I tried to use a key but that does not seem to be picked up either.

**Comment by tumon-art** [comment]
I'm also facing same issue 
`Please, log in from the web page. Waiting...
Error: lookup localhost on 8.8.8.8:53: no such host`

**Comment by JeremyStarTM** [comment]
I have this problem too, using LibreWolf.
`Please, log in from the web page. Waiting...
Error: lookup localhost on 1.1.1.1:53: no such host`

**Comment by tumon-art** [comment]
✔️  It works when I connect to the VPN

**Comment by JeremyStarTM** [comment]
I have a solution: Just add this to your /etc/hosts:

`127.0.0.1 localhost`

That line was missing and that's why it did not work. No VPN required.
